<ui version="4.0" >
 <class>WallpaperWindow</class>
 <widget class="QMainWindow" name="WallpaperWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>463</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>524</width>
    <height>463</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Masaüstü Arkaplanı - Anatolya Başlangıç</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>442</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="WallpapersGroup" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title" >
     <string>Arkaplanlar</string>
    </property>
    <widget class="QListWidget" name="listWallpapers" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="showPictureGroup" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>10</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title" >
     <string>Önizleme</string>
    </property>
    <widget class="QLabel" name="pictureLabel" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>40</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextButton" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>İleri</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/go-next/go-next.png</normaloff>:/go-next/go-next.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Geri</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/go-previous/go-previous.png</normaloff>:/go-previous/go-previous.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton" >
    <property name="geometry" >
     <rect>
      <x>430</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>İptal</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/exit-button/application-exit.png</normaloff>:/exit-button/application-exit.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>310</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text" >
     <string>Masaüstü arkaplanınızı seçin.
Bilgi: Çok renkli arkaplanlar gözünüzü yorabileceği için siyah tonlarına yakın
arkaplanlar seçmenizi öneririz.</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>442</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../icons/startericons.qrc" />
 </resources>
 <connections/>
</ui>
